vllm/model_executor/layers/fused_moe/pplx_dispatch_combine.py

```diff
@@ -0,0 +1,118 @@
+# SPDX-License-Identifier: Apache-2.0
+from typing import List, Optional, Tuple
+
+import pplx_kernels as pplx
+import torch
+
+import vllm.model_executor.layers.fused_moe.modular_kernel as mk
+from vllm.model_executor.layers.fused_moe.utils import _fp8_quantize
+
+
+# Note use: layer.get_all_to_all() to get an AllToAll instance
+# The max_num_tokens, world_size and dp_size must be the same
+# as the ones used to create the AllToAll.  Unfortunately, there's
+# no way(?) to extract this info from AllToAll
```

Comment:
>Unfortunately, there's no way(?) to extract this info from AllToAll

Worth putting up a PR to address this?

Reply:
Yeah, I could do that.  It would be handy.

Reply:
https://github.com/vllm-project/collab-perplexity/pull/2

Reply:
Could you open the PR here https://github.com/ppl-ai/pplx-kernels instead?